### PR TITLE
Test against Ruby 2.6

### PR DESCRIPTION
**Pull request by segiddins**
# Description:

Since 2.6 has been released, we should make sure we support it!

# Tasks:

- [ ] Describe the problem / feature
- [ ] Write tests
- [ ] Write code to solve the problem
- [ ] Get code review from coworkers / friends

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ environment:
   - ruby_version: 23-x64
   - ruby_version: 24-x64
   - ruby_version: 25-x64
+  - ruby_version: 26-x64
```
Comment (segiddins) [inline appveyor.yml]: ruby didn't seem to get installed?

File: .travis.yml [hunk 2]
```diff
@@ -12,14 +12,15 @@ rvm:
   - 2.3.8
   - 2.4.5
   - 2.5.3
+  - 2.6.0
```
Comment (segiddins) [inline .travis.yml]: Bundler is still failing on 2.6, working on that over in https://github.com/bundler/bundler/pull/6856

File: appveyor.yml (hunk 1 above)
Comment (MSP-Greg) [reply appveyor.yml]: @segiddins

With normal CI, all Ruby builds (except trunk) used on Appveyor are pre-installed.

1. The RubyInstaller2 2.6.0 build is not passing due to a last minute update to BigDecimal, which changed spec tests (added to ruby/ruby, not yet added to ruby/spec).

2. Once released, Appveyor may take a few days (or weeks) to add the build to their images.

3. Trunk/head is not affected.

4. I'll ping when it's been added...

**Comment by colby-swandale** [comment]
@segiddins what are you thoughts are leaving 2.6 out of appveyor for the time being so that we can merge this PR?

**Comment by MSP-Greg** [comment]
See 810671e

Appveyor has not yet updated their images with Ruby 2.6.x.  They will be updating with 2.6.1, along with installing both Bundler 1.7.x & 2.x.

**Comment by MSP-Greg** [comment]
@segiddins 

Appveyor has added Ruby 2.6.1.

JFYI, Bundler versions 1.17.3 & 2.0.1 are install on Ruby versions 2.3 & later.  At present, trunk/head only has what's included in the Rubygems std-lib.

See:
https://ci.appveyor.com/project/MSP-Greg/appveyor-ruby/builds/22294299#L908

**Comment by hsbt** [comment]
@bundlerbot r+

**Pull request merged**